### ISSUE TITLE
luaposix in gentoo  repo

### ISSUE THREAD

**Issue by bedroge** [EESSI/gentoo-overlay#9]
Gentoo's repository now includes a new luaposix as well (same version as ours: 35.0), so maybe we should stick to their version?

https://bugs.gentoo.org/show_bug.cgi?id=603812
https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=2513f4e45d12860e82dbe2d2022acc83d7842b82

**Comment by bedroge** [comment]
I tried to use their luaposix and bit32, but it still fails for me:
```
>>> Install dev-lua/lua-bit32-5.3.5 into /cvmfs/pilot.eessi-hpc.org/compat/x86_64/var/tmp/portage/dev-lua/lua-bit32-5.3.5/image
>>> Completed installing dev-lua/lua-bit32-5.3.5 into /cvmfs/pilot.eessi-hpc.org/compat/x86_64/var/tmp/portage/dev-lua/lua-bit32-5.3.5/image

 * Final size of build directory: 316 KiB
 * Final size of installed tree:   32 KiB

 * QA Notice: /cvmfs/pilot.eessi-hpc.org/compat/x86_64//cvmfs/pilot.eessi-hpc.org/compat/x86_64/ double prefix
 * QA Notice: /cvmfs/pilot.eessi-hpc.org/compat/x86_64//cvmfs/pilot.eessi-hpc.org/compat/x86_64/usr double prefix
 * QA Notice: /cvmfs/pilot.eessi-hpc.org/compat/x86_64//cvmfs/pilot.eessi-hpc.org/compat/x86_64/usr/lib64 double prefix
 * QA Notice: /cvmfs/pilot.eessi-hpc.org/compat/x86_64//cvmfs/pilot.eessi-hpc.org/compat/x86_64/usr/lib64/lua double prefix
 * QA Notice: /cvmfs/pilot.eessi-hpc.org/compat/x86_64//cvmfs/pilot.eessi-hpc.org/compat/x86_64/usr/lib64/lua/5.1 double prefix
 * QA Notice: /cvmfs/pilot.eessi-hpc.org/compat/x86_64//cvmfs/pilot.eessi-hpc.org/compat/x86_64/usr/lib64/lua/5.1/bit32.so double prefix
 * ERROR: dev-lua/lua-bit32-5.3.5::gentoo failed:
 *   Aborting due to QA concerns: double prefix files installed
 * 
 * Call stack:
 *   misc-functions.sh, line 596:  Called install_qa_check
 *   misc-functions.sh, line 132:  Called source 'install_symlink_html_docs'
 *            05prefix, line 114:  Called install_qa_check_prefix
 *            05prefix, line  19:  Called die
 * The specific snippet of code:
 *   		die "Aborting due to QA concerns: double prefix files installed"
 * 
```

**Comment by bedroge** [comment]
This will be fixed in #18.
